### PR TITLE
feat: ib registry adapter beta fixes [WEB-632] [WEB-642] [WEB-671]

### PR DESCRIPTION
**Pull request by gotzenx**

### CODE REVIEW

**Review by lbertenasco — DISMISSED** [review]
Approving since solidity looks ok, but I have no idea of what it does

**Review by geewynn — APPROVED** [review]

File: contracts/RegistryAdapters/IronBankRegistryAdapter.sol [hunk 1]
```diff
@@ -658,11 +670,12 @@ contract RegistryAdapterIronBank is Ownable {
             currentPositionIdx++;
         }
         if (borrowBalanceShares > 0) {
+            uint256 borrowedCyTokenBalance = (borrowBalanceShares * 10**18) / exchangeRateStored;
```
Comment (jstashh) [inline contracts/RegistryAdapters/IronBankRegistryAdapter.sol:673]:
```suggestion
            uint256 borrowedUnderlyingTokenBalance = (borrowBalanceShares * 10**18) / exchangeRateStored;
```

File: contracts/RegistryAdapters/IronBankRegistryAdapter.sol [hunk 2]
```diff
@@ -658,11 +670,12 @@ contract RegistryAdapterIronBank is Ownable {
...
             positions[currentPositionIdx] = Position({
                 assetId: assetAddress,
                 tokenId: tokenAddress,
                 typeId: positionBorrow,
-                balance: borrowBalanceShares,
+                balance: borrowedCyTokenBalance,
```
Comment (jstashh) [inline contracts/RegistryAdapters/IronBankRegistryAdapter.sol:678]:
```suggestion
                balance: borrowedUnderlyingTokenBalance,
```

File: contracts/RegistryAdapters/IronBankRegistryAdapter.sol [hunk 3]
```diff
@@ -641,7 +653,7 @@ contract RegistryAdapterIronBank is Ownable {
 
         if (supplyBalanceShares > 0) {
             uint256 supplyBalanceUnderlying =
-                (supplyBalanceShares * asset.exchangeRateStored()) / 10**18;
+                (supplyBalanceShares * exchangeRateStored) / 10**18;
```
Comment (x48115) [inline contracts/RegistryAdapters/IronBankRegistryAdapter.sol:656]: Stylistic change

**Review by x48115 — DISMISSED** [review]

**Pull request merged**